app/Http/routes.php

```diff
@@ -11,12 +11,50 @@
 |
 */
 
-Route::get('/', function () {
-    $missed_calls = DB::table('missed_calls')->orderBy('created_at', 'desc')->get();
-    return view('welcome', ["missed_calls" => $missed_calls]);
-});
+/**
+ * Main view
+ */
+Route::get(
```

Comment:
The welcome page is a commodity not referenced in the tutorial, and not meant to be tested. So we could tolerate this.